BedrockServer.cpp

```diff
@@ -1231,10 +1220,6 @@ BedrockServer::~BedrockServer() {
     }
     SINFO("Threads closed.");
 
-    if (_outstandingSocketThreads) {
-        SWARN("Shutting down with " << _outstandingSocketThreads << " socket threads remaining.");
```

Comment:
Why remove this log line..?

Reply:
This got removed in an intermediate change where this was impossible to reach. I can add it back.